src/services/aragon-backend/queries/use-daos.ts

```diff
@@ -63,24 +65,27 @@ const fetchDaos = async (
 };
 
 export const useDaos = (
-  params: IFetchDaosParams,
-  options: UseInfiniteQueryOptions<IPaginatedResponse<IDao>> = {}
+  initialParams: IFetchDaosParams,
```

Comment:
Let's keep the same names for infinite and normal queries
```suggestion
  params: IFetchDaosParams,
```